packages/@react-spectrum/s2/stories/TableView.stories.tsx

```diff
@@ -931,6 +932,35 @@ export const ColSpan: StoryObj<typeof ColSpanExample> = {
   }
 };
 
+
+export const TableWithTextFields: StoryObj<typeof TableView> = {
+  render: (args) => (
+    <TableView aria-label="Many items table" {...args} styles={style({width: 800, height: 400})}>
+      <TableHeader columns={manyColumns}>
+        {(column) => (
+          <Column allowsResizing minWidth={100} isRowHeader={column.name === 'Column 1'}>{column.name}</Column>
+        )}
+      </TableHeader>
+      <TableBody items={manyRows}>
+        {item => (
+          <Row id={item.id} columns={manyColumns}>
```

Comment:
why so many rows and columns? We should try to make a realistic example

Reply:
It's funny, I've actually seen a screenshot of one which looked much like this (though only one textfield to a cell, but i wanted more for testing)
But I'm happy to change it to something a little more real life looking.